joe_test.go

```diff
@@ -229,8 +229,11 @@ data: world
 func TestJoe_errors(t *testing.T) {
 	t.Parallel()
 
+	fin, err := sse.NewFiniteReplayProvider(1, false)
+	tests.Equal(t, err, nil, "should create new FiniteReplayProvider")
```

Comment:
This is now failing because we check that capacity is greater or equal to 2
```suggestion
	fin, err := sse.NewFiniteReplayProvider(2, false)
	tests.Equal(t, err, nil, "should create new FiniteReplayProvider")
```